crmsh/bootstrap.py

```diff
@@ -669,11 +669,11 @@ def init_cluster_local():
     if pass_msg:
         logger.warning("You should change the hacluster password to something more secure!")
 
-    start_pacemaker()
+    start_pacemaker(enable=True)
     wait_for_cluster()
 
 
-def start_pacemaker(node_list=[]):
+def start_pacemaker(node_list=[], enable=False):
```

Comment:
I tend to remove `enable` for the argument, and also remove it when calling utils.start_service().
To make the strong position that `start_pacemaker()` has nothing related to systemd `enable`.